compiler/main/version.cpp

```diff
@@ -31,7 +31,7 @@ void
 get_version(char *v) {
   v += sprintf(v, "%d.%s.%s", MAJOR_VERSION, MINOR_VERSION, UPDATE_VERSION);
   if (strcmp(BUILD_VERSION, "0") != 0 || developer)
-    sprintf(v, " pre-release (%s)", BUILD_VERSION);
+    sprintf(v, ".%s", BUILD_VERSION);   // post-release
```

Comment:
I'm a little disappointed to find that when I implemented this, I didn't make the printing or the pre-release-specific information configurable by flipping a single PRE_RELEASE #define or somesuch...  I thought I'd made the two formats coexist more than it seems I did...  Good intentions...